ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json

```diff
@@ -339,112 +339,113 @@
             "SubmitButton": "[es]OK"
         },
         "MoreDetails": {
-            "Title": "[es]Learn more about the Metastatic Breast Cancer Project",
+            "Title": "Aprende más sobre el Metastatic Breast Cancer Project",
             "MainText": {
                 "Section1": {
-                    "Pt1": "[es]The Metastatic Breast Cancer Project takes a new approach to cancer research in which researchers partner directly with patients, who share their samples and clinical information in order to speed important discoveries. The MBCproject is part of",
-                    "Pt2": "[es]a nonprofit organization that brings together patients and researchers as partners to accelerate discoveries in cancer research.",
-                    "Pt3": "[es]is stewarded by four leading organizations: Emerson Collective, a California-based social change organization; the Broad Institute of MIT and Harvard, a leading nonprofit biomedical research institution; the Biden Cancer Initiative, an independent nonprofit organization that builds on the federal government’s Cancer Moonshot; and the Dana-Farber Cancer Institute, a leading cancer hospital."
+                    "Pt1": "El Metastatic Breast Cancer Project es un nuevo modelo en la investigación del cáncer en el cual los investigadores colaboran directamente con pacientes, quienes comparten sus muestras de tumor e información clínica para juntos acelerar descubrimientos. El MBCproject es parte de",
```

Comment:
I'd suggest replace `Project` with `proyecto`

Reply:
I'm not sure about that... I use the provided translation, probably it was important to keep the original name of the project? as you can notice other words in the name are not translated as well

Reply:
Ok, let's keep it as is. 
Perhaps, translators will take a look at the completed work and polish smth.